agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/RuntimeConfiguration.java

```diff
@@ -22,4 +22,10 @@ public class RuntimeConfiguration {
   public String instrumentationLoggingLevel;
 
   public String selfDiagnosticsLevel;
+
+  public Configuration.PreviewConfiguration preview = new Configuration.PreviewConfiguration();
+
+  public Configuration.ProfilerConfiguration profiler = new Configuration.ProfilerConfiguration();
```

Comment:
to make it clear that we don't support dynamic configuration of all Preview things, only:
```suggestion
  public boolean profilerEnabled;
```